builder/pointer.go

```diff
@@ -52,6 +52,43 @@ func (*Pointer) Build(gen Generator, ctx *MethodContext, sourceID *xtype.JenID,
 	return stmt, xtype.VariableID(jen.Id(outerVar)), err
 }
 
+// SourcePointer handles type were only the source is a pointer.
+type SourcePointer struct{}
+
+// Matches returns true, if the builder can create handle the given types.
+func (*SourcePointer) Matches(source, target *xtype.Type) bool {
+	return source.Pointer && !target.Pointer
+}
+
+// Build creates conversion source code for the given source and target type.
+func (*SourcePointer) Build(gen Generator, ctx *MethodContext, sourceID *xtype.JenID, source, target *xtype.Type) ([]jen.Code, *xtype.JenID, *Error) {
```

Comment:
This behavior should not be enabled by default, it should be behind a converter comment. See https://github.com/jmattheis/goverter/issues/11 for my reasoning.

Reply:
Thank your for giving understanding why it is important. Let me improve this.

Reply:
Added machinery to check status of the every converter by the method context. All converters but the `SourcePointer` are enabled by default. SourcePointer checks corresponding method flag.